T2CC_APP/t2cc/app/src/main/java/com/example/t2cc/RegistrationActivity.java

```diff
@@ -227,7 +243,32 @@ private boolean validateRegistrationForm() {
       mPasswordField.setError("Required.");
       mPasswordField.requestFocus();
       valid = false;
-    } else {
+    } else if (password.length() < 10) {
+      mPasswordField.setError("Password needs to be at least 10 char. ");
```

Comment:
Remove extra space at end of sentence.
Change to "Password must contain at least 10 characters."